st2common/st2common/util/crypto.py

```diff
@@ -17,8 +17,38 @@
 
 
 def symmetric_encrypt(encrypt_key, message):
+    """
+    Encrypt the given message using the encrypt_key. Returns a UTF-8 str
+    ready to be stored in database. Note that we convert the hex notation
+    to a ASCII notation to produce a UTF-8 friendly string.
+
+    Also, this method will not return the same output on multiple invocations
+    of same method. The reason is that the Encrypt method uses a different
+    'Initialization Vector' per run and the IV is part of the output.
+
+    :param encrypt_key: Symmetric AES key to use for encryption.
+    :type encrypt_key: :class:`keyczar.keys.AesKey`
```

Comment:
for this method does it matter if its an AesKey? Is there is higher interface/superclass?